about.html

```diff
@@ -641,68 +661,55 @@ <h3 class="sitemap-title">COMPANY</h3>
       </section>
 
       <section class="footer-copyright">
-        <h2 style="display: none">copyright</h2>
-        <p>BINABOX 2022 - ALL rights reserved</p>
+        <h2>BINABOX 2022 - ALL rights reserved</h2>
```

Comment:
I think no reason to use `h2 `tag in footer, `p` is much better

Reply:
I agree, but I'm also confused ) 
validator.nu shows warning - Section lacks heading. Consider using h2-h6 elements to [add identifying headings to all sections](https://www.w3.org/wiki/HTML/Usage/Headings/Missing).

And is it a good practice to add `<h2 style="display: none">copyright</h2>` hidden h2-6 to sections without heading?
Or we can ignore those warnings?

I have decided to fix this by changing `section` to `div` to avoid redundant hidden h2.

` <div class="footer-copyright">
        <p>BINABOX 2022 - ALL rights reserved</p>`

#fixed until further clarification

Reply:
Your solution with `div` instead of `section` much better than hidden div. 
Using <h2 style="display: none">copyright</h2> not good practice to fix validator warning.